src/screens/MainApp/DiagnoseRequest/screens/ImageReview/components/ConfirmButton/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { PrimaryButton, GrayButton } from '~/components/buttons'
+import { Description } from '~/components/texts'
+import styles from './styles'
+
+const ConfirmButton = ({ errorState, onConfirm }) => (
+  <>
+    {errorState
+      ? (
+        <GrayButton
+          style={styles.button}
+        >
+          <Description white>Confirmar</Description>
+        </GrayButton>
+      )
+      : (
```

Comment:
I don't know if Standard says anything about it, but I like this in one line: `) : (`, it's just the same as a `} else {`.